recipes/newrelic-telemetry-sdk-python/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "newrelic-telemetry-sdk" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/newrelic-telemetry-sdk-{{ version }}.tar.gz
+  sha256: 461ba12a54f3c44e85e89343d99f66228b4d4703cd2808cefc8fc0b83c97e4e7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5,!=3.6
+    - setuptools-scm
+    - pip
+  run:
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5,!=3.6
```

It is now wrong but, b/c we dropped py27 a long time ago, this is easier to read and equivalent to the many "!" there.

```suggestion
    - python >=3.6
    - setuptools-scm
    - pip
  run:
    - python >=3.6
```